src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -175,20 +196,66 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-@use "../../../../styles/variables";
-@use "../../../../styles/mixins";
+@use '../../../../styles/variables';
+@use '../../../../styles/mixins';
+@use '../../../../styles/typography';
 
 .qr-code-reader {
-  .icon {
-    color: variables.$color-blue;
+  .qr-scan-wrapper {
+    @include mixins.flex(center, center);
+
+    background-color: variables.$color-bg-1;
+    border: 4px solid rgba(variables.$color-white, 0.05);
+    border-radius: 50%;
+    width: 56px;
+    height: 56px;
+    align-self: center;
+    margin-bottom: 16px;
+    margin-top: 44px;
+
+    .qr-scan {
+      margin-bottom: 0;
+      width: 40px;
+      height: 40px;
+      color: variables.$color-blue;
+    }
+  }
+
+  .button-wrapper {
+    padding-top: 20px;
+    padding-bottom: 28px;
+    width: 100%;
   }
 
-  .camera video {
-    max-width: 70vw;
+  .camera {
+    video {
```

Comment:
It is considered as a bad pattern to style HTML elements. Please use CSS classes.